packages/test/test-service-load/src/nodeStressTest.ts

```diff
@@ -102,10 +102,19 @@ async function orchestratorProcess(
         undefined,
         args.browserAuth);
 
-    // Create a new file if a testId wasn't provided
-    const url = args.testId !== undefined
-        ? await testDriver.createContainerUrl(args.testId)
-        : await initialize(testDriver, seed, profile, args.verbose === true);
+    // If testId is provided, then try to load the file first.
+    let url;
+    if (args.testId !== undefined) {
+        const docExists: boolean = await testDriver.doesDocumentExists(args.testId);
+        if (docExists) {
+            url = await testDriver.createContainerUrl(args.testId);
```

Comment:
i don't think we need a new function. just encode this behavior in createContainerUrl and return undefined in the case where it doesn't exit